src/signTx.c

```diff
@@ -666,6 +735,12 @@ static void signTx_handleInput_ui_runStep()
 
 	UI_STEP_BEGIN(ctx->ui_step, this_fn);
 
+	UI_STEP(HANDLE_INPUT_STEP_DISPLAY) {
+		char headerText[18];
+		snprintf(headerText, SIZEOF(headerText), "Input #%u", BODY_CTX->currentInput);
```

Comment:
Please discuss with Rafael whether inputs are indexed from 0 or 1 in user interfaces, and check a block explorer, too. (the same for collateral)